### ISSUE TITLE
0.2.2 cannot find project files on Linux

### ISSUE THREAD

**Issue by mauve** [eisnstein/PackCheck#1]
Hi,

Thanks for a great tool. Since the last update I have had problems with paths on Linux though, a mismatch between \ and /. The `dotnet` cli tooling handles that somehow magically and it worked in version 0.1.1 of `packcheck` but seems to be broken in 0.2.2.

`dotnet sln add tools/cli/cli.csproj` creates the following entry in the solution file: `Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "cli", "tools\cli\cli.csproj", "{FAC77532-FF57-48E8-B482-85F55462E2EB}"`

Then invoking `packcheck` prints the following error:

```
13:03 $ packcheck
WARN: File tools\cli\cli.csproj does not exist in the current directory /home/mikae/repos/api
```

**Comment by eisnstein** [comment]
Hi @mauve, thanks for the report. Sorry for the late response, I will have a look at it these days.

**Comment by eisnstein** [comment]
@mauve I just published version 0.2.3 which should fix the issue.
